base/src/Data/Macaw/Memory/ElfLoader.hs

```diff
@@ -752,16 +752,6 @@ relaTargetARM64 end msegIndex symtab rel addend relFlag =
                          , relocationEndianness = end
                          , relocationJumpSlot   = True
                          }
-    Elf.R_AARCH64_GLOB_DAT -> do
```

Comment:
@joehendrix was this case intended to be `Elf.R_AARCH64_TLS_DTPMOD64` or is it just a spurious duplicate?
```
[ 8 of 30] Compiling Data.Macaw.Memory.ElfLoader ( src/Data/Macaw/Memory/ElfLoader.hs, /project/macaw/dist-newstyle/build/x86_64-linux/ghc-8.10.1/macaw-base-0.3.15.1/build/Data/Macaw/Memory/ElfLoader.o, /project/macaw/dist-newstyle/build/x86_64-linux/ghc-8.10.1/macaw-base-0.3.15.1/build/Data/Macaw/Memory/ElfLoader.dyn_o )

src/Data/Macaw/Memory/ElfLoader.hs:755:5: error: [-Woverlapping-patterns, -Werror=overlapping-patterns]
    Pattern match is redundant
    In a case alternative: Elf.R_AARCH64_GLOB_DAT -> ...
    |
755 |     Elf.R_AARCH64_GLOB_DAT -> do
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
```

Reply:
That looks like a duplicate -- probably from a copy paste error.  Good call.